mne/preprocessing/tests/test_stim.py

```diff
@@ -39,11 +39,12 @@ def test_fix_stim_artifact():
     tmin_samp = int(-0.035 * epochs.info['sfreq']) - e_start
     tmax_samp = int(-0.015 * epochs.info['sfreq']) - e_start
 
-    epochs = fix_stim_artifact(epochs, tmin=tmin, tmax=tmax, mode='linear')
-    data = epochs.get_data()[:, :, tmin_samp:tmax_samp]
-    diff_data0 = np.diff(data[0][0])
-    diff_data0 -= np.mean(diff_data0)
-    assert_array_almost_equal(diff_data0, np.zeros(len(diff_data0)))
+    for mode in ['linear', 'cubic']:
+        epochs = fix_stim_artifact(epochs, tmin=tmin, tmax=tmax, mode=mode)
+        data = epochs.get_data()[:, :, tmin_samp:tmax_samp]
+        diff_data0 = np.diff(data[0][0])
+        diff_data0 -= np.mean(diff_data0)
+        assert_array_almost_equal(diff_data0, np.zeros(len(diff_data0)))
```

Comment:
this diff makes sense for linear interp. If this passes it means cubic interp does a linear interp in the end.